docs/index.md

```diff
@@ -1,33 +1,35 @@
 # Overview
 
-Investin is the first decentralized fund management protocol live on Solana and Binance Smart Chain that facilitates trustless interactions between prospecting investors and aspiring traders, the protocol provides an open market for investors and traders to exchange their offering in a trustless and non-custodial environment i.e. investors keep custody of their assets at all times, while traders make investment decisions and manage assets with the goal of producing a return for both parties.
+Investin is the first Decentralized Asset Management protocol live on Solana and BSC that transforms aspiring traders into Fund Managers by providing them a turstless and non-custodial platform as a service for active management of capital. This enables investors to earn passive returns on the capital invested in the funds whilst retaining the custody of their assets all along. 
 
 # On-chain visibility
 
- The trading history, positions and Assets Under Management (AUM) of Investin Fund Managers are always transparent to Investors. Investor decision making about joining, leaving or staying with a particular fund is informed by having access to real-time and historical on-chain data. 
+ The trading history, positions, active Assets Under Management (AUM) and returns performance of Investin's Funds are always transparent. Investors are empowered to make an informed decision before investing, withdrawaing or staying invested in a particular fund by having access to real-time and historical on-chain data. 
+
+ Fund Managers who showcase consistent perfomance on Investin and outperform other managers in the market, are able to access more capital to trade with, by attracting new investments and generating higher returns, enhancing the protocol's growth and user adoption.
 
- Fund Managers can’t hide their positions, performance can’t be fudged and any deviations from a funds investment strategy are visible to investors straight away. Imagine a Fund Manger is highly leveraged beyond their advertised strategy, an investor can easily vote with their feet by exiting the fund. Fund Managers who show expected perfomance can build a community using their reputation on Investin which can increase inflows of new liquidity from investors.
 ## Protocol Users
 
 * Managers/Traders:
 
-The driving force and future owners of the protocol that run funds to attract as many investors to their funds to generate a stable income for both parties. 
+The driving force and future owners of Investin, who actively manage assets on their Funds to generate returns on the invested capital and attract further investments.
```

Comment:
The driving force and future owners of Investin. They actively manage assets of their Funds to generate returns on the invested capital and attract future investments.